docs/content/usage/cli/examples.md

```diff
@@ -77,6 +77,15 @@ lego --email="foo@bar.com" --http --csr=/path/to/csr.pem run
 
 (lego will infer the domains to be validated based on the contents of the CSR, so make sure the CSR's Common Name and optional SubjectAltNames are set correctly.)
 
+### Obtain a certificate of type rsa instead of the default ec keys
+
+Elliptic curve keys are not supported by old systems, so you want to fall back on rsa:
```

Comment:
RSA and EC are acronyms, please change to uppercase.

I would also like to specify what "old systems" are, or at least link to more information. GlobalSign has a neat overview [here](https://support.globalsign.com/ssl/ssl-certificates-life-cycle/ecc-compatibility). (The footnotes didn't survive being moved between CMS, from the looks of it though. It would be nicer, if something similar would exist on letsencrypt.org.)

From what I can gather, you'll need an RSA key, if you absolutely must support hard- or software which has reached or is near EOL. The only non-EOL entry on the list at GlobalSign is CentOS/RHEL, with the minimum version required being 6.5 (released in 2014).